source/agent/addons/internalIO/InternalClientWrapper.h

```diff
@@ -0,0 +1,50 @@
+// Copyright (C) <2021> Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef INTERNALCLIENTWRAPPER_H
+#define INTERNALCLIENTWRAPPER_H
+
+#include "../../addons/common/MediaFramePipelineWrapper.h"
+#include <InternalClient.h>
+#include <logger.h>
+#include <nan.h>
+
+/*
+ * Wrapper class of owt_base::InternalClient
+ */
+class InternalClient : public FrameSource,
+                       public owt_base::InternalClient::Listener {
+public:
+    DECLARE_LOGGER();
+    static NAN_MODULE_INIT(Init);
+
+    owt_base::InternalClient* me;
+    boost::mutex mutex;
+    std::queue<std::string> stats_messages;
```

Comment:
Better include <queue> in "include" section as you're directly using this here. (Though  currently does not break build).